keras/engine/topology.py

```diff
@@ -1088,7 +1088,7 @@ def add_loss(self, losses, inputs=None):
         if hasattr(self, '_losses'):
             self._losses += losses
         # Update self._per_input_updates
-        if inputs == []:
+        if inputs is []:
```

Comment:
fix it by add a check for list type. please let me know if you have any comments about it.